src/main.js

```diff
@@ -135,14 +141,35 @@ function randomizePoster() {
   mainImage.src = images[getRandomIndex(images)];
   mainTitle.innerText = titles[getRandomIndex(titles)];
   mainQuote.innerText = quotes[getRandomIndex(quotes)];
+  currentPoster = new Poster(
+    mainImage.src,
+    mainTitle.innerText,
+    mainQuote.innerText
+  );
 }
 
 function toggleMakePoster() {
-  viewMainPoster.classList.toggle('hidden');
-  viewMakeOwnPoster.classList.toggle('hidden')
+  viewMainPoster.classList.toggle("hidden");
+  viewMakeOwnPoster.classList.toggle("hidden");
 }
 
 function toggleShowSavedPoster() {
-  viewMainPoster.classList.toggle('hidden');
-  viewShowSavedPosters.classList.toggle('hidden')
+  viewMainPoster.classList.toggle("hidden");
+  viewShowSavedPosters.classList.toggle("hidden");
```

Comment:
Consistency is important for your code standards. I'm sure you'll learn about linting soon - a linter will help improve consistency around your code.